src/clojure_tic_tac_toe/input_output.clj

```diff
@@ -50,6 +98,12 @@
         :else move))))
 
 
+(defn have-the-computer-move
+  [board & args]
+  (println "\nThe computer moved.")
+  (computer_move/make-random-move board))
```

Comment:
This doesn't seem to belong here

Reply:
I included it in `input_output.clj` because it does some output (`"The computer moved."` message).

Reply:
That's line 103, right? Does line 104 belong here?

Reply:
I guess it makes sense to have this full function somewhere else. I kept in `input_output.clj` initially, because the other "move" function was there also. I'll see where I can put this.